test/offsetarray.jl

```diff
@@ -268,8 +268,7 @@ am = map(identity, a)
 # other functions
 v = OffsetArray(v0, (-3,))
 @test_approx_eq v v
-@test parent(v') == v0'
```

Comment:
I'm happy to do that, but the point of the test seemed to be to test how offset vectors perform transposition, but now that behavior is "owned" by `RowVector` and types don't have to make sure this is implemented anymore (even if it used to be via `similar` or whatever).